skrub/_gap_encoder.py

```diff
@@ -23,18 +22,164 @@
 from sklearn.utils.extmath import row_norms, safe_sparse_dot
 from sklearn.utils.validation import _num_samples, check_is_fitted
 
-from ._utils import check_input
+from . import _dataframe as sbd
+from ._on_each_column import SingleColumnTransformer
 
 
-class GapEncoderColumn(BaseEstimator, TransformerMixin):
-    """GapEncoder for encoding a single column.
+class GapEncoder(SingleColumnTransformer, TransformerMixin):
+    """Constructs latent topics with continuous encoding.
```

Comment:
the docstring has been moved essentially unchanged from the previous GapEncoder with 2 minor changes:

- the docstring still mentioned 'empty_impute', but apparently the parameter has been renamed 'zero_impute'
- the docstring mentioned 'inverse_transform' but it doesn't seem to be defined

Reply:
I assume that this was resolved after what we discuss in the meeting.